packages/wethegit-components/src/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react"
+
+import styles from "./accordion.module.scss"
+import AccordionItem, { CardsData } from "./component/AccordionItem"
+import cards from "./content.js"
```

Comment:
I think content should be passed to the component as a prop and not shipped with it, otherwise you can't reuse it.